src/interpreter/scheme/base.rs

```diff
@@ -245,10 +245,9 @@ fn vector_ref<R: RealNumberInternalTrait, E: IEnvironment<R>>(
 ) -> Result<Value<R, E>> {
     let mut iter = arguments.into_iter();
     match iter.next() {
-        None => logic_error!("vector_ref requires exactly two argument"),
+        None => logic_error!("vector-ref requires exactly two argument"),
```

Comment:
Since it was checked before call native function. should we just write an `unreachable!()`?